benchmarking/commons/hpo_main_sagemaker.py

```diff
@@ -83,11 +83,11 @@ def parse_args(
             dict(
                 name="warm_pool",
                 type=int,
-                default=1,
+                default=0,
```

Comment:
Reverting an earlier change. It is probably better to have warm pools switched off by default, because they require a bit of additional effort to get going.